src/validation.cpp

```diff
@@ -1056,10 +1063,10 @@ bool MemPoolAccept::AcceptSingleTransaction(const CTransactionRef& ptx, ATMPArgs
 /** (try to) add transaction to memory pool with a specified acceptance time **/
 static bool AcceptToMemoryPoolWithTime(const CChainParams& chainparams, CTxMemPool& pool, TxValidationState &state, const CTransactionRef &tx,
                         int64_t nAcceptTime, std::list<CTransactionRef>* plTxnReplaced,
-                        bool bypass_limits, const CAmount nAbsurdFee, bool test_accept) EXCLUSIVE_LOCKS_REQUIRED(cs_main)
+                        bool bypass_limits, const CAmount nAbsurdFee, bool test_accept, CAmount* fee_out=nullptr) EXCLUSIVE_LOCKS_REQUIRED(cs_main)
```

Comment:
please move the fee arg before the test arg, so that non-testing callers can use this as well (without having to specify that they don't test)

Reply:
Done.

Reply:
Also i was wondering cant a non-testing call use something `(..., false, &fee)` to use the `fee` while passing `false` in `test_accept`?

Reply:
Ok, fair enough. No need to change if it is too involved.